src/views/Settings.vue

```diff
@@ -146,8 +139,7 @@
             {{ translate('View shipping orders along with pickup orders.') }}
           </ion-card-content>
           <ion-item lines="none">
-            <ion-label>{{ translate("Show shipping orders") }}</ion-label>
-            <ion-toggle :checked="showShippingOrders" @ionChange="setShowShippingOrdersPreference($event)" slot="end" />
+            <ion-toggle label-placement="start" :checked="showShippingOrders" @ionChange="setShowShippingOrdersPreference($event)">{{ translate("Show shipping orders") }}</ion-toggle>
```

Comment:
Update methods to remove the extra check that was added to not update the value when updated programatically

Reply:
@amansinghbais Handle this case for methods where we have used ionChange event